mlflow/spark.py

```diff
@@ -573,6 +616,19 @@ def _validate_model(spark_model):
         )
 
 
+def _is_spark_connect_model(spark_model):
+    """
+    Return whether the spark model is spark connect ML model
+    """
+    try:
+        from pyspark.ml.connect import Model as ConnectModel
+
+        return isinstance(spark_model, ConnectModel)
+    except ImportError:
+        # pyspark < 3.5 does not support connect ML model
```

Comment:
Which one is the correct name?

- Spark ML connect model
- Spark connect model
- Spark connect ML model

Reply:
I think we should use name `Spark connect ML model`.

Reply:
Got it, let's be consistent.